Source/DafnyLanguageServer/Workspace/IdeState.cs

```diff
@@ -139,4 +161,379 @@ public record IdeState(
   public IEnumerable<Uri> GetDiagnosticUris() {
     return StaticDiagnostics.Keys.Concat(VerificationResults.Keys);
   }
+
```

Comment:
I assume all the code from this point on is relocated rather than brand new? If not, how's the testing coverage?

Reply:
It is relocated, yes